libraries/radpdfprocessing/formats-and-conversion/convert-to-image/using-image-format-provider.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Using SkiaImageFormatProvider
+page_title: Using SkiaImageFormatProvider
+slug: radpdfprocessing-formats-and-conversion-image-using-skiaimageformatprovider
+tags: pdf,image,skia
+published: True
+position: 1
+---
+
+# Using SkiaImageFormatProvider
+
+Since R3 2022 the __RadPdfProcessing__ library support converting entire documents to images. This is achieved by using the third party library SkiaSharp. You can convert to various formats, using synchronous or asynchronous export. 
+
+>important This feature is only available in the NET Standard version of the suite. For other versions check the fllowing article: [ThumbnailFactory](https://docs.telerik.com/devtools/wpf/controls/radpdfviewer/features/export-fixedpage-to-image)
+>
+
+## Required packages
+
+* The __Telerik.Documents.Fixed.FormatProviders.Image.Skia__ assembly.
+* The __SkiaSharp__ Nuget package. 
+* The __SkiaSharp.NativeAssets.*__ Nuget package. This package may differ according to the used platform. There are version for Windows, MacOs, Linux, WebAssembly, Android, iOS, and others.
+
+## Using the SkiaImageFormatProvider
+
+To convert your documents to images use the __Export__ method. Please nota that the export method does not accept a document but a page. This is why you need to iterate all pages. In this example each page is saved in a separate file.  
```

Comment:
To convert your documents' pages to images, use the __Export__ method. Note that the export method does not accept a document but a page. This is why you need to iterate all pages. In this example, each page is saved in a separate file.